tensorrt_llm/_torch/auto_deploy/custom_ops/triton_kernels/rope.py

```
@@ -2,6 +2,191 @@
 
 
+@triton.jit
+def rope_fwd_interleaved_kernel(
+    q_ptr,  # [B, S, H_Q, D] input Q (interleaved layout)
+    k_ptr,  # [B, S, H_K, D] input K (interleaved layout)
+    cos_cache_ptr,  # [max_seq_len, D] cos cache
+    sin_cache_ptr,  # [max_seq_len, D] sin cache
+    position_ids_ptr,  # [B, S] position IDs
+    q_out_ptr,  # [B, S, H_Q, D] output Q
+    k_out_ptr,  # [B, S, H_K, D] output K
+    B,  # batch size
```

Ruff (0.14.14)

[warning] Unused function argument: `B`

(ARG001)